plugin/src/main/java/io/fabric8/maven/plugin/mojo/develop/LogMojo.java

```diff
@@ -40,7 +42,7 @@
 
     @Override
     protected void applyEntities(Controller controller, final KubernetesClient kubernetes, final String namespace, String fileName, final Set<HasMetadata> entities) throws Exception {
-        tailAppPodsLogs(kubernetes, namespace, entities, false, null, followLog, null, true);
+        new PodLogService(getLogServiceContext()).tailAppPodsLogs(kubernetes, namespace, entities, false, null, followLog, null, true);
```

Comment:
I wonder whether the PodLogService could be already put into the base class, so you don't have to expose the PodLogServiceContext but just a `AbstractTailLogMojos/getPodLogService()` ?